tests/headers.test.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as core from '@actions/core';
+import { expect } from 'chai';
+import 'mocha';
+import * as sinon from 'sinon';
+import { parseHeadersInput } from '../src/headers';
+
+/**
+ * Unit Test parseHeadersInput method in headers.
```

Comment:
Lets also add integration tests in https://github.com/google-github-actions/upload-cloud-storage/blob/main/tests/upload.int.test.ts that verify default behavior and custom headers

Reply:
I added `uploads a single file with metadata` integration test.